scripts/install/qt_mac_installer.sh

```diff
@@ -73,7 +73,7 @@ done
 # Render the plugins relative to the executable:
 cd $WEBOTS_HOME/lib/webots/qt/plugins
 
-declare -a libs=("imageformats/libqjpeg.dylib" "platforms/libqcocoa.dylib" "styles/libqmacstyle.dylib" "tls/libqsecuretransportbackend.dylib" "tls/libqcertonlybackend.dylib")
+declare -a libs=("imageformats/libqjpeg.dylib" "platforms/libqcocoa.dylib" "styles/libqmacstyle.dylib" "tls/libqsecuretransportbackend.dylib" "tls/libqcertonlybackend.dylib" "tls/libqopensslbackend.dylib")
```

Comment:
Is this a new dependency of Qt that we need to add here? If yes, I wonder if we should not mention (even briefly) that this case could arise and what to do in this case on the wiki page: https://github.com/cyberbotics/webots/wiki/Qt-compilation.